Frameworks/include/UITouchInternal.h

```diff
@@ -17,29 +17,31 @@
 
 #import <UIKit/UITouch.h>
 
-@class WUIPointerPoint;
-@class WUXIPointerRoutedEventArgs;
+#include "COMIncludes.h"
+#import <winrt/Windows.UI.Input.h>
+#import <winrt/Windows.UI.Xaml.Input.h>
+#include "COMIncludes_End.h"
 
 @interface UITouch () {
 @public
     // Adding a WUXIPointerRoutedEventArgs property so that we can mark the routed event as handled/not handled
     // when touchsBegan:withEvent method is called.
-    StrongId<WUXIPointerRoutedEventArgs> _routedEventArgs;
+    TrivialDefaultConstructor<winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs> _routedEventArgs;
     NSTimeInterval _timestamp;
     float _touchX, _touchY;
     float _previousTouchX, _previousTouchY;
     unsigned _tapCount;
     UITouchPhase _phase;
     StrongId<UIView> _view;
-    WUIPointerPoint* _pointerPoint; // Only valid during the processing of a pointer event; don't try to retain it.
+    TrivialDefaultConstructor<winrt::Windows::UI::Input::PointerPoint> _pointerPoint;
```

Comment:
>TrivialDefaultConstructor [](start = 4, length = 25)

I'm a little confused why this is needed. Is this because PointerPoint doesn't have a default ctor in the cpp winrt types?